### DISCUSSION TITLE
experimentalRunAllSpecs: When new tab is opened instead of closing and reopening the browser, I am running into the memory not enough issue.

### DISCUSSION THREAD

**Discussion by MaNoJTaMaNG1997** [cypress-io/cypress#25211] [General]
Previously, when I used to run multiple specs at once, the browser used to close and reopen, but after the recent fixes for **experimentalRunAllSpecs: Replace with new tab when selecting specs rather than closing browser #24919** the specs are run in a new tab, but this usually leads to the issue of browsers memory not enough issue. Is there any workaround it. And is anybody else facing the same issue? So, now I have to run all the specs one by one to prevent the browser running into memory not enough issue.

**Comment by lmiller1990** [comment]
Hi, thanks for reporting this. Sounds more like a bug than a discussion... but let me get some more info first. How many specs/tests are you running at once? Can you estimate specs (number of files) tests (number of `it`) and ~ time to run?

Also can you share your system - OS, memory, that sort of thing? Thanks.

It sounds like we might have a memory issue we need to fix.

**Reply by lmiller1990** [reply]
I made an issue: https://github.com/cypress-io/cypress/issues/25218 for better visibility. Can you share the answer to ^^^ there? Thanks!
